Rebuild-Black-Business/RBB-Website
# Fix/pagination links need unique keys

# Describe your PR
This PR fixes the `unique "key" prop` warning you get when a list items have the same key value, or in this case, the root node did not have the `key` prop, it was on the child node, so I just moved it and that fixes the error.

Fixes: [add-keys-to-pagination](https://trello.com/c/47zXNuW9/221-add-keys-to-pagination) trello ticket

## Pages/Interfaces that will change
No UI changes

### Screenshots / video of changes
Before:  was seeing `Warning: Each child in a list should have a unique "key" prop. Check the render method of 'Pagination'. See https://fb.me/react-warning-keys for more information.`

After: no longer seeing that^ error in console

## Steps to test

1. Go to /businesses 
2. Open console
3. Scroll towards the bottom to see `Warning: Each child in a list should have a unique "key" prop. Check the render method of 'Pagination'. See https://fb.me/react-warning-keys for more information.`
4. Pull this PR down and you no longer will see that ^ warning


### Additional notes
